OpenRA.Mods.Common/Activities/FindResources.cs

```diff
@@ -103,7 +103,7 @@ public override Activity Tick(Actor self)
 				foreach (var n in notify)
 					n.MovingToResources(self, closestHarvestablePosition.Value, this);
 
-				return ActivityUtils.SequenceActivities(mobile.MoveTo(closestHarvestablePosition.Value, 1), new HarvestResource(self), this);
+				return ActivityUtils.SequenceActivities(mobile.MoveTo(closestHarvestablePosition.Value, 2), new HarvestResource(self), this);
```

Comment:
I've already added a mobile.CanEnterCell check to that calculation in a test branch locally, the problem is that 'CanEnter' still needs to be re-evaluated right before moving onto that cell, because it might have been free when `closestHarvestablePosition` ran but blocked by the time the harvester gets there.